deps/minimatch/src/dist/cjs/assert-valid-pattern.d.ts

```diff
@@ -0,0 +1,2 @@
+export declare const assertValidPattern: (pattern: any) => void;
```

Comment:
> (Or the .map files - which by the way - do they work when debugging node core when using this?)

I believe they do work if you `--enable-source-maps`, but it might be best to just work out how to tell TS to build in a more "naive" way, and pull the .d.ts files into `./typings` somewhare.  If the MJS build can work in node core, then that'd be ideal, since it's essentially the same exact code as the TS, but without type annotations.  The CJS build is a bit more crufty.

(No opinion about build process impact, that's y'all's call.)

Reply:
I have just taken the library as is without changes, like most other dependencies